packages/open-next/src/adapters/plugins/README.md

```diff
@@ -0,0 +1,15 @@
+### Known issues
```

Comment:
@conico974 I spent a couple hours trying to figure out why the replacement stuff was broken... Looks like an esbuild bug removing the top comment when it omits the types, it also drops the head comment :|

Reply:
I see, i didn't test with `import type`. But do we really need to put those inside the override ? esbuild will remove them anyway, can't we just put them at the top of the file ?

Reply:
Good point - yes we can. I'll update the known issue to note that `types` should be excluded from the plugin comments.